ui/ducks/send/send.js

```diff
@@ -1354,14 +1358,25 @@ const slice = createSlice({
             tokens,
             tokenAddressList,
             isProbablyAnAssetContract,
+            udTlds,
           } = action.payload;
-
+          let validUDDomain = false;
+          try {
+            udTlds.payload.forEach((tld) => {
+              if (state.recipientInput.toLowerCase().endsWith(`.${tld}`)) {
+                validUDDomain = true;
+              }
+            });
+          } catch {
```

Comment:
I don't believe this code will throw an error? perhaps if state.recipientInput is undefined but in no other cases that I can think of? Maybe just a optional chaining call if its possible that state.recipientInput may be undefined in this case and drop the try/catch?

Reply:
we were getting an error on the e2e tests and then the try catches seemed to resolve them, but we can investigate